packages/relayer/src/index.ts

```diff
@@ -5,6 +5,8 @@ import { WalletConfig } from '@0xsequence/config'
 import { proto } from './rpc-relayer'
 
 export interface Relayer {
+  // simulate returns the execution results for a list of transactions.
+  simulate(wallet: string, ...transactions: Transaction[]): Promise<TransactionResult[]>
```

Comment:
should we rename `TransactionResult` to `TxnSimulateResult` or `SimulateResult` ..?